bin/lib/zopen-build

```diff
@@ -518,123 +500,6 @@ setDepsEnv()
 
 setEnv()
 {
-  # Set up the compiler and option variables
-  # CC - the C compiler
```

Comment:
where did all the setenv code go?

Reply:
That is in the .env of the comp_* dependencies - https://github.com/ZOSOpenTools/comp_xlclangport/blob/main/buildenv#L32